assets/less/app.less

```diff
@@ -1,5 +1,5 @@
 @import 'npm://font-awesome/less/font-awesome';
-@import 'npm://highlightjs/styles/tomorrow';
+@import 'npm://highlight.js/src/styles/tomorrow';
```

Comment:
If we keep the license in the generated app.js file, we can remove them from license. Otherwise we will need to copy them all.